restapi.js

```diff
@@ -0,0 +1,170 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const mongoose = require('mongoose')
+
+
+const {User, Transaction} = require('./model')
+
+const app = express()
+
+app.use(bodyParser.json())
+
+const port = 4500;
+
+const DBUri = "mongodb://localhost:27017/Users"
+
+mongoose.connect(DBUri)
+.then(() => {
+    console.log(`Database is connected`)
+})
+.catch(() => {
+    console.log(`Database is not connected`)
+});
+
+// Create an account
+app.post('/Createaccount/', async (req, res) => {
```

Comment:
avoid using CAPS in your API url's

Reply:
Done

Reply:
Yes you have done some but there is still some left, I already made another comment on that area.